transformation/src/test/test-files/outputs/library_prep_activity/part-1.json

```diff
@@ -0,0 +1,134 @@
+{"biosample_id":"ENCBS051RNA","id":"ENCLB179ZZZ","library_id":"ENCLB179ZZZ","library_material":"SO:0000356","library_material_name":"RNA"}
```

Comment:
Duplicating the library ID here is kinda weird / wasteful. We _could_ drop that column and just use the `library_id` as the PK.

I'm also unsure of the general usefulness of this table vs. just adding the two extra columns to the `library` table. I get that conceptually there are 2 different things being stored in 2 different tables, but it's not clear to me if ENCODE exposes enough info about the activity for it to be useful as a separate table.

Reply:
Oops, I missed this comment when I made my comment below...

Reply:
I definitely agree that it might make sense to combine them... Maybe a good thing to get Kathy's thoughts on if we meet up to talk about Encode tomorrow morning?